.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/_nuts/github.com/ThomsonReutersEikon"]
+	path = vendor/_nuts/github.com/ThomsonReutersEikon
+	url = https://github.com/ThomsonReutersEikon/go-ntlm
```

Comment:
As a side note, the contributing doc outlines how to add deps with nut, if you hadn't seen it. I always have to consult it because I never remember.